patroni/dcs/etcd.py

```diff
@@ -613,18 +639,23 @@ def wrapper(self, *args, **kwargs):
 
 class Etcd(AbstractEtcd):
 
-    def __init__(self, config):
+    def __init__(self, config: Dict[str, Any]) -> None:
         super(Etcd, self).__init__(config, EtcdClient, (etcd.EtcdLeaderElectionInProgress, EtcdRaftInternal))
         self.__do_not_watch = False
 
-    def set_ttl(self, ttl):
+    @property
+    def _client(self) -> EtcdClient:
+        assert isinstance(self._abstract_client, EtcdClient)
+        return self._abstract_client
+
+    def set_ttl(self, ttl: int) -> Optional[bool]:
         self.__do_not_watch = super(Etcd, self).set_ttl(ttl)
```

Comment:
Suggest adding explicit `return None` to appease my IDE!

Reply:
Weird. Which IDE do you use?